core/modules/admin_bar/css/admin_bar.css

```diff
@@ -360,11 +217,25 @@
 
 /* Second-and-more-level hovering. */
 #admin-bar .dropdown li li.expandable > a {
-  background: #45454A url(../images/arrow.png) no-repeat 145px center; /* LTR */
+  background: #45454A;
+}
+#admin-bar .dropdown li li.expandable > a::before {
+  content: '';
+  /*noinspection CssUnresolvedCustomProperty*/
+  mask-image: var(--icon-arrow-square-right); /* LTR */
```

Comment:
I think `--icon-arrow-right` should be used instead. At that small size, it's hard to make out the arrow with a square around it. And it looks too busy.

Reply:
I swapped out this icon and I agree it does look a lot simpler and less noisy.